lib/Runtime/Library/JavascriptBigInt.cpp

```diff
@@ -0,0 +1,216 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+#include "RuntimeLibraryPch.h"
+
+namespace Js
+{
+    Js::Var JavascriptBigInt::Initialize(const char16 * content, charcount_t cchUseLength, bool isNegative, ScriptContext * scriptContext)
```

Comment:
Two minor things here:
1. For static methods that allocate and return objects, we almost always call them `Create`
2. I'd prefer to return the most specific type we can (`JavascriptBigInt*`). If the caller needs a `Var` then they can implicitly convert (without needing any reinterpret_cast) because `Var` is `void*` and all pointer types can be implicitly turned into `void*`. So I expect that `Var v = JavascriptBigInt::Create(...)` would compile successfully, for example.